frontend/src/ui/Footer/Footer.tsx

```diff
@@ -3,8 +3,8 @@ import styles from './Footer.module.css';
 const Footer: React.FunctionComponent = () => {
 	return (
 		<footer className={styles.container}>
-			Built with 🖤 by the Elon Society of Computing Project Team in Partnership
-			with the Elon University Office of IT
+			{/* Built with 🖤 by the Elon Society of Computing Project Team in Partnership
```

Comment:
```suggestion
			{/* Built with ❤️ by the Elon Society of Computing Project Team in Partnership
```

Reply:
Do we have to reserve the footer for the a display of partnership with the Elon University Office of IT? If not, a footer may not be necessary and we can add in in the docs or put it in the hero

Reply:
This isn't a negotiated requirement but to put forth that we're building in partnership with the institution does lead to our credbility.